ssv/runner.go

```diff
@@ -83,14 +114,19 @@ func (b *BaseRunner) baseConsensusMsgProcessing(runner Runner, msg *qbft.SignedM
 		prevDecided, _ = b.State.RunningInstance.IsDecided()
 	}
 
+	if err := b.processPreConsensusJustification(runner, b.highestDecidedSlot, msg); err != nil {
+		return false, nil, errors.Wrap(err, "invalid pre-consensus justification")
+	}
+
 	decidedMsg, err := b.QBFTController.ProcessMsg(msg)
 	if err != nil {
 		return false, nil, err
 	}
 
 	// we allow all consensus msgs to be processed, once the process finishes we check if there is an actual running duty
+	// do not return error if no running duty
 	if !b.hasRunningDuty() {
-		return false, nil, errors.New("no running duty")
+		return false, nil, nil
```

Comment:
can you please explain this change?

Reply:
joining the request

Reply:
per the comment in code

 we allow all consensus msgs to be processed, once the process finishes we check if there is an actual running duty
do not return error if no running duty